Makefile

```diff
@@ -67,10 +69,9 @@ compile: \
 compile: ## compile jsonnet sources to json
 .PHONY: compile
 
-down: \
-	clean
 down: ## remove stack (warning: it removes the database container)
-	@$(COMPOSE) down
+	# FIXME: docker-compose down fails because of potsie network attached to elastic search
+	@$(COMPOSE) down || echo WARNING: unable to remove the stack
```

Comment:
Is it a network related issue? AFAIR I did not encounter this issue.

Reply:
In my case this happens because `elasticsearch` depends on the `potsie` network and `docker-compose down` tries to remove it.

Reply:
A solution is to manually stop ralphs `elasticsearch` container and then run `make down` on potsie. Will update the comment)